src/styles/blocks/catalog.scss

```diff
@@ -12,14 +14,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
It would be great to create a variable for transition duration, as it should be the same for the whole page